release_notes/ocp-4-0-release-notes.adoc

```diff
@@ -0,0 +1,711 @@
+horizontal pod autoscaling (HPA) based on any metric provided by Prometheus.
+Autoscale based on any cluster-level metrics {product-title} provides out of the
+box and autoscale based on any app metrics.
+
+Limitations:
+
+The adapter only connects to a single Prometheus instance (or a set of
+load-balanced replicas, using Kubernetes services). Manually deploying adapter
+and configuring it to use Prometheus (that includes Prometheus instance shipped
+in OpenShift) For using cluster-metrics, a cluster administrator needs to
+whitelist metrics that can be used to configure HPA.
+
+[[ocp-4-0-developer-experience]]
+=== Developer experience
+
+[[ocp-4-0-code-ready-containers]]
+==== Code ready containers
+
+A local desktop instance of {product-title} 4.0 replaces the functions of `oc
+cluster` commands, Minishift, and CDK. {product-title} 4.0 focuses on ease of
+access and native experience, with a native installer on macOS and Microsoft
+Windows, native hypervisor support, and tray icon integration.
+
+[[ocp-4-0-registry]]
+=== Registry
+
+[[ocp-4-0-networking]]
+=== Networking
+
+[[ocp-4-0-multus]]
+==== Multus (Technology Preview)
+
+Multus is now available in xref:ocp-4-0-technology-preview[Technology Preview]
+and not for production workloads.
+
+Multus is a meta plug-in for Kubernetes Container Network Interface (CNI), which
+enables a user to create multiple network interfaces per pod.
+
+[[ocp-4-0-master]]
+=== Master
+
+[[ocp-4-0-web-console]]
+=== Web console
+
+[[ocp-4-0-developer-catalog]]
+==== Developer Catalog
+
+{product-title} 4.0 features a redesigned Developer Catalog that brings all of
+the new Operators and existing broker services together, with new ways to
+discover, sort ,and understand how to best use each type of offering. The
+Developer Catalog is the entry point for a developer to access all services
+available to them. It merges all capabilities from Operators, the Service
+Catalog, brokers, and Source-to-Image (S2I).
+
+[[ocp-4-0-operator-powered-UI-extensions]]
+==== Operator-powered user interface extensions
+
+Now with Operators, you can pick and choose what functionality you want to
+enable, allowing the user to customize the interface.
+
+[[ocp-4-0-new-management-screens]]
+==== New management screens
+
+New management screens in {product-tite} 4.0  support automated operations.
+Examples include the management of taints, tolerations, and cluster settings.
+
+[[ocp-4-0-security]]
+=== Security
+
+In {product-title} 4.0, an Operator is utilized to install, configure, and manage
+the certificate signing server. Certificates are managed as secrets, and are
+stored and encrypted in etcd.
+
+[[ocp-4-0-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 4.0 introduces the following notable technical changes.
+
+
+[discrete]
+[[ocp-4-0-SecurityContextConstraints]]
+==== SecurityContextConstraints
+
+`SecurityContextConstraints` now only exist in the `security.openshift.io` group.
```

Comment:
@deads2k Can you please help elaborate on this point? What does this mean for our users?